src/main/kotlin/org/elaastic/questions/player/PlayerController.kt

```diff
@@ -696,7 +699,7 @@ class PlayerController(
         val sequence = sequenceService.get(id, true)
         val chatGptEvaluation = chatGptEvaluationService.findEvaluationById(evaluationId)
 
-                // Check authorizations
+        // Check authorizations
         requireAccessThrowDenied(user == chatGptEvaluation!!.response.learner) {
             "You must be the learner of the response to report the evaluation"
```

Comment:
This message should be localised.

Reply:
fix in 09e686797934b5b93215be3c23cb797531a07f9e
and  23b2612ed38c667809ebbf82bd3b340eb2fc74e5